content/22.vaccines.md

```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
 
 ### Live-Attenuated Viruses
 
-<!--To Do: Not used this time I think?-->
-Although this is the vaccine development strategy with the longest history... (short paragraph explaining not really in play here)
+Live-attenuated vaccines (LAV) or replication-competent vaccines use a weakened living version of a disease-causing virus or a version of a virus that is similar to induce an immune response [@doi:10.1038/s41586-020-2798-3]. 
+The virus can be attenuated by passinging it in a foreign host, until as a consequence of selection pressure the virus loses its efficacy in the original host. 
+Alternatively, selective gene deletion or codon de-optimization can be utilized to attenuate the virus [@doi:10.1038/s41586-020-2798-3]. 
+LAVs are used globally to prevent diseases caused by viruses such as measles, rubella, polio, influenza, varicella zoster, and the yellow fever virus [@doi:10.1016/B978-0-323-35761-6.00002-X].
+It is generally recognized that LAVs induce an immune response that is similar to natural infection and are favoured because they induce long-lasting and robust immunity that can protect from disease. 
+This is in part as a result of the immune response to the range of viral antigens available that tend to be more immunogenic than those from non-replicating vaccines [@url:https://www.hhs.gov/immunization/basics/types/index.html; @doi:10.1016/j.virol.2015.03.032; @doi:10.3389/fimmu.2020.602256]. 
```

Comment:
I found this sentence a little hard to follow -- would these edits keep the same meaning?
```suggestion
This strong protective effect is induced in part by the immune response to the range of viral antigens available from LAV, which tend to be more immunogenic than those from non-replicating vaccines [@url:https://www.hhs.gov/immunization/basics/types/index.html; @doi:10.1016/j.virol.2015.03.032; @doi:10.3389/fimmu.2020.602256]. 
```